src/main/scala/aserralle/akka/stream/kcl/Errors.scala

```diff
@@ -4,14 +4,9 @@
 
 package aserralle.akka.stream.kcl
 
-import scala.util.control.NoStackTrace
-
 object Errors {
-
-  sealed trait KinesisWorkerSourceError extends NoStackTrace
```

Comment:
Can we get back this `KinesisWorkerSourceError` as a marker trait? Removing it is an unnecessary API change.

Reply:
can you check again